spiped/dispatch.c

```diff
@@ -125,6 +127,8 @@ callback_gotconn(void * cookie, int s)
 
 	/* We have gained a connection. */
 	A->nconn += 1;
+	if (A->only_one)
```

Comment:
This changes the behaviour of the `-1` option: Previously we would accept multiple connections and exit as soon as the any of them was closed; this makes spiped stop accepting connections once one has arrived.

@gperciva, that option was something you added for testing purposes, I think... does this affect your testing?  I don't mind changing the behaviour of the option given that it's documented as "for internal testing", but I don't want to break the tests...